src/main/java/com/mraof/minestuck/network/GristEntityPacket.java

```diff
@@ -0,0 +1,52 @@
+package com.mraof.minestuck.network;
+
+import com.mraof.minestuck.entity.item.GristEntity;
+import net.minecraft.client.Minecraft;
+import net.minecraft.network.FriendlyByteBuf;
+import net.minecraft.world.entity.Entity;
+
+import java.util.Objects;
+
+public class GristEntityPacket implements PlayToClientPacket
```

Comment:
The packet name isn't specific enough to convey the purpose of the packet

Reply:
fixed